networking/kvm.go

```diff
@@ -420,6 +424,7 @@ func kvmTransformFlannelNetwork(net *activeNet) error {
 	net.conf.Type = n.Delegate["type"].(string)
 	net.conf.IPMasq = n.Delegate["ipMasq"].(bool)
 	net.conf.MTU = n.Delegate["mtu"].(int)
+	net.conf.IsDefaultGateway = n.Delegate["isDefaultGateway"].(bool)
```

Comment:
@jellonek Done via https://github.com/coreos/rkt/pull/2769/files#diff-c8c480671fc15dd96fce4719a423d7c3R540
since IsGW is a field in BridgeNetConf